applications/asset_tracker/src/main.c

```diff
@@ -1377,7 +1377,7 @@ void cloud_event_handler(const struct cloud_backend *const backend,
 		LOG_INF("CLOUD_EVT_READY");
 		ui_led_set_pattern(UI_CLOUD_CONNECTED);
 
-#if defined(CONFIG_BOOTLOADER_MCUBOOT)
+#if defined(CONFIG_BOOTLOADER_MCUBOOT) && !IS_ENABLED(CONFIG_NRF_CLOUD_FOTA)
```

Comment:
use `defined` `IS_ENABLED` macro AFAIK is reserved for using it in a C `if` and not preprocessor.

The reason for using IS_ENABLED in a regular `if` is that you can catch function signature changes etc. The code path will be compiled but not linked.